docs/source/reference/integration.rst

```diff
@@ -1,3 +1,5 @@
+.. module:: optuna.integration
+
 .. _integration_list:
```

Comment:
Now `.. _integration_list` is not necessary now. I think it better to remove this and update https://github.com/optuna/optuna/blob/d809aacd0384a17d06dcab03cf99d5c4c86abc92/tutorial/007_pruning.py#L79.